### ISSUE TITLE
Figure 1 not open

### ISSUE THREAD

**Issue by Cizzy5** [StevenMHernandez/ESP32-CSI-Tool#104]
When I run Python csi live.py with esp idf, Python not responding.
![IMG-20240331-WA0003](https://github.com/StevenMHernandez/ESP32-CSI-Tool/assets/165564347/bf338ab8-b660-4db5-a1d1-e8b2b151e6a9)

**Comment by StevenMHernandez** [comment]
@Cizzy5, do you see any CSI printed with the standard `idf.py monitor`?

@AsciiPro99, any suggestions for why there is not any CSI plotted?

**Comment by AsciiPro99** [comment]
If you can print CSI values to monitor fine with the above command. It is likely due to missing imports. Confirm Stevens inquiry and then post your package list.

**Comment by Cizzy5** [comment]
Yes i see CSİ PRİNTED with standard idf.py monitor, i also visualized csi
data by Matlab, but when I run live Python code not open matplotlib figure.
Thanks for you interested.

6 Nis 2024 Cmt ÖS 4:28 tarihinde Steven M. Hernandez <
***@***.***> şunu yazdı:

**Comment by AsciiPro99** [comment]
Could you post your package list?
